02_calculator/calculator.rb

```diff
@@ -1 +1,16 @@
-#write your code here
+def add(num1, num2)
+	num1 + num2
+end
+
+def subtract(num1, num2)
+  num1-num2
+end
+
+def sum(array)
+  array.sum
+end
+ 
+def multiply(num1, num2)
+  num1 * num2
+end
+
```

Comment:
Some method blocks are missing here, make sure you have `power` && `factorial` blocks so u can pass all RSpec tests.